### PR TITLE
fix(endorsement-system): Removing unused code for party letter and party application

### PR DESCRIPTION
**Pull request by alexandradilja**
Removing unused code for party letter and party application.
All unused code regarding these two applications has been removed. The goal of this is to reduce build errors andto make the endorsement system a little less complicated.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by arnorhs — APPROVED** [review]
LGTM! 

image

**Review by darrikonn — APPROVED** [review]
Core looks good... love seeing code removals :D!

**Review by thordurhhh — APPROVED** [review]
Service-portal LGTM.

**Review by eirikurn — APPROVED** [review]
Nice!

File: apps/services/endorsements/api/migrations/20210325113632-create-initial-tables.js [hunk 1]
```diff
@@ -25,10 +25,6 @@ module.exports = {
       tags: {
         type: Sequelize.ARRAY(Sequelize.STRING),
       },
-      validation_rules: {
-        type: Sequelize.JSONB,
-        defaultValue: '[]',
-      },
       owner: {
```
Comment (pshomov) [inline apps/services/endorsements/api/migrations/20210325113632-create-initial-tables.js:28]: In general we cannot be modifying existing migrations. Unless this is not a real migration but some kind of hack I kind of recall we were using to create data.

File: apps/services/endorsements/api/migrations/20210811145737-endorsement-list-metadata-sync.js [hunk 2]
```diff
@@ -18,61 +18,7 @@ module.exports = {
...
-                'partyApplicationSudurkjordaemi2021',
-                'partyApplicationSudvesturkjordaemi2021',
-              ],
-              'varchar[]',
-            ),
-          },
-        },
-        { transaction: t },
-      )
     })
```
Comment (pshomov) [inline apps/services/endorsements/api/migrations/20210811145737-endorsement-list-metadata-sync.js:22]: Same comment as the other migration.

**Comment by pshomov** [comment]
The schema generation is failing due to some missing file, probably needs fixing.

File: apps/services/endorsements/api/migrations/20210325113632-create-initial-tables.js (hunk 1 above)
Comment (alexandradilja) [reply apps/services/endorsements/api/migrations/20210325113632-create-initial-tables.js:28]: Ok, that makes sense! I'll add it back in.

**Pull request merged**